flags.go

```diff
@@ -33,7 +33,7 @@ const (
 	// maxSequentialReadSizeMb is the max value supported by sequential-read-size-mb flag.
 	maxSequentialReadSizeMb = 1024
 	// DefaultStatCacheCapacity is the default value for stat-cache-capacity.
-	DefaultStatCacheCapacity = 4096
+	DefaultStatCacheCapacity = 1 << 20 // 1048576 = 1024x1024 = 1Mi
```

Comment:
Assuming, we have discussed over this default change with Marco.

Reply:
Yes, discussed with Marco and other stakeholders in a meeting earlier.